packages/inline-banner/addon/templates/components/nucleus-inline-banner.hbs

```diff
@@ -1,9 +1,23 @@
 {{#if hasBlock}}
-  {{yield
+  <div class="nucleus-inline-banner__icon">
+    {{nucleus-icon name=_icon size="medium" variant=type}}
+  </div>
+  <div class="nucleus-inline-banner__content">
+    {{yield
     (hash
       close=(action "onCloseTip")
```

Comment:
We need not yield the close action anymore, since it's already going to be there. Just `yield` should suffice.

Reply:
Okay, i will do it.